grpc/request.go

```diff
@@ -0,0 +1,47 @@
+package grpc
+
+import (
+	"github.com/gojek/fiber"
+	"github.com/gojek/fiber/protocol"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/proto"
+)
+
+type Request struct {
+	// Metadata will hold the grpc headers for request
+	Metadata metadata.MD
+	// RequestPayload is the grpc request
+	RequestPayload proto.Message
```

Comment:
nit: instead of `RequestPayload` it's better to name it only `Payload`. It's more idiomatic in Go to not repeat the field name with its container's name.

Reply:
rename to message as interface method is `Payload()`